kolibri/plugins/coach/assets/src/views/learners/LearnerSummaryPage/LearnerHeader.vue

```diff
@@ -8,7 +8,7 @@
           :text="$tr('back')"
         />
       </p>
-      <ReportsControls />
+      <ReportsControls @export="exportCSV" />
```

Comment:
Not sure, but the issue Peter noted about the download starting multiple times might be fixed w/ `@export.prevent="exportCSV"` or something similar.

Reply:
Hi Jacob! this does not seem to be working. `Error in v-on handler: "TypeError: Cannot read properties of undefined (reading 'preventDefault')"`.

Reply:
Is it that the download is starting multiple times, or is it that there are two files that were causing that pop up? I think if it's the two files, and we use the design approach that Jessica suggests above, that will resolve the issue by "splitting up" the file download buttons. If there's something that's happening in the code side that is triggering the same export multiple times, this approach could work.

Reply:
Hi Marcella! I was only checking for the case when the CSV export button was downloading 2 files simultaneously. We won't face this with two different buttons.